src/main/resources/com/google/api/codegen/csharp/sample.snip

```diff
@@ -124,18 +124,18 @@
                         foreach ({@class.pageStreaming.resourceElementTypeName} {@class.pageStreaming.resourceVarName} in response.{@class.pageStreaming.resourceFieldName})
                         {
                             // TODO: Change code below to process each `{@class.pageStreaming.resourceVarName}` element:
-                            Console.WriteLine("[" + {@class.pageStreaming.resourceVarName}.Key + "] = " + {@class.pageStreaming.resourceVarName}.Value);
+                            Console.WriteLine(JsonConvert.SerializeObject({@class.pageStreaming.resourceVarName}));
```

Comment:
It may be better to keep the distinction between Key and Value, to make it clear these are always expected to be present on the result elements.

I would guess that the key would normally be a primitive type (e.g. string) that should print correctly, so maybe:
`Console.WriteLine("[" + {@class.pageStreaming.resourceVarName}.Key + "] = " + JsonConvert.SerializeObject({@class.pageStreaming.resourceVarName}.Value));`